x/lsnative/staking/keeper/liquid_stake.go

```diff
@@ -0,0 +1,158 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/persistenceOne/persistence-sdk/v2/x/lsnative/staking/types"
+)
+
+// SetTotalLiquidStakedTokens stores the total outstanding tokens owned by a liquid staking provider
+func (k Keeper) SetTotalLiquidStakedTokens(ctx sdk.Context, tokens sdk.Int) {
+	store := ctx.KVStore(k.storeKey)
+
+	tokensBz, err := tokens.Marshal()
+	if err != nil {
+		panic(err)
+	}
+
+	store.Set(types.TotalLiquidStakedTokensKey, tokensBz)
+}
+
+// GetTotalLiquidStakedTokens returns the total outstanding tokens owned by a liquid staking provider
+// Returns zero if the total liquid stake amount has not been initialized
+func (k Keeper) GetTotalLiquidStakedTokens(ctx sdk.Context) sdk.Int {
+	store := ctx.KVStore(k.storeKey)
+	tokensBz := store.Get(types.TotalLiquidStakedTokensKey)
+
+	if tokensBz == nil {
+		// QUESTION: Should we panic here instead?
```

Comment:
If we are adding a migration for this key, then panic makes sense here I guess.

Reply:
Done